src/pymatgen/symmetry/analyzer.py

```diff
@@ -274,7 +274,10 @@ def _get_symmetry(self) -> tuple[NDArray, NDArray]:
             "translations" gives the numpy float64 array of the translation
             vectors in scaled positions.
         """
-        dct = spglib.get_symmetry(self._cell, symprec=self._symprec, angle_tolerance=self._angle_tol)
```

Comment:
For this we get:
```
tests/analysis/chemenv/connectivity/test_structure_connectivity.py::TestStructureConnectivity::test_serialization
  /Users/yang/developer/pymatgen/venv312/lib/python3.12/site-packages/spglib/spglib.py:640: DeprecationWarning: Use get_magnetic_symmetry() for cell with magnetic moments.
    warnings.warn(
```

But this doesn't look like a in place replace, for:
```python
dct = spglib.get_magnetic_symmetry(self._cell, symprec=self._symprec, angle_tolerance=self._angle_tol)
```

I got 6 test failure:
```

tests/symmetry/test_analyzer.py ..F....FF.F..........F.F..........                                                                             [100%]

====================================================================== FAILURES ======================================================================
__________________________________________ TestSpacegroupAnalyzer.test_get_conventional_standard_structure ___________________________________________
```